skimage/metrics/_adapted_rand_error.py

```diff
@@ -20,12 +20,14 @@ def adapted_rand_error(image_true=None, image_test=None, *, table=None,
     ignore_labels : sequence of int, optional
         Labels to ignore. Any part of the true image labeled with any of these
         values will not be counted in the score.
+    alpha : float, optional
+       Relative weight given to precision and recall in the adapted Rand error
+       calculation.
```

Comment:
```suggestion
        Relative weight given to precision and recall in the adapted Rand error
        calculation.
```
just aligning the tabulation ;)